src/plugins/network/RequestDetails.js

```diff
@@ -532,6 +532,7 @@ class JSONTextFormatter {
   format = (body: string, contentType: string) => {
     if (
       contentType.startsWith('application/json') ||
+      contentType.startsWith('application/hal+json') ||
```

Comment:
This stuff here seems unrelated. Could you split this up into a separate PR? :)

Reply:
I split into two pr, please review again